wolfcrypt/src/aes.c

```diff
@@ -10643,6 +10643,11 @@ static WARN_UNUSED_RESULT int roll_auth(
     word32 remainder;
     int ret;
 
+    /* Sanity check on authIn to prevent segfault in xorbuf() where
+     * variable 'in' is dereferenced as the mask 'm' in misc.c */
+    if (in == NULL)
```

Comment:
We should be checking authIn is not NULL when authInSz > 0 in wc_AesCcmEncrypt().
Don't want to have parameter checks in internal functions if possible.

Reply:
Fixed, thank you!